src/renderers/dom/shared/__tests__/ReactMount-test.js

```diff
@@ -297,18 +295,6 @@ describe('ReactMount', () => {
     expect(calls).toBe(5);
   });
 
-  it('tracks root instances', () => {
-    // Used by devtools.
-    expect(Object.keys(ReactMount._instancesByReactRootID).length).toBe(0);
-    ReactTestUtils.renderIntoDocument(<span />);
-    expect(Object.keys(ReactMount._instancesByReactRootID).length).toBe(1);
-    var container = document.createElement('div');
-    ReactDOM.render(<span />, container);
-    expect(Object.keys(ReactMount._instancesByReactRootID).length).toBe(2);
-    ReactDOM.unmountComponentAtNode(container);
-    expect(Object.keys(ReactMount._instancesByReactRootID).length).toBe(1);
-  });
-
```

Comment:
Oh, we should just move the backend here. It's still on my todo list but I haven't gotten around to it.